packages/grid/_modules_/grid/hooks/features/rows/gridEditRowsSelector.ts

```diff
@@ -0,0 +1,3 @@
+import { GridState } from '../core/gridState';
+
+export const gridEditRowsStateSelector = (state: GridState) => state.editRows;
```

Comment:
This is the base selector, future selector for edit rows will be built on top of this one, using createSelector

Reply:
When I was renaming the public APIs there were some that were defined but never used (at least internally). I would also suggest if it isn't used to remove it. We should spend some time auditing and remove what is unused.

Reply:
My concern is with having a base selector in the first place. When you are working on the data grid, and you want to refactor something, or even understand how the state is used, you have to search for two things: the base selector, and how the sector is used. Both seem to use different naming conventions in the codebase; I couldn't easily understand how the state was used. Making it simpler to follow seems important.

I'm asking my previous question because it wasn't answered. What's the upside of having two selectors for accessing one value in the state?

I don't understand it. I can only see the one downside around making it harder for new developers to jump into the data grid, contributing improvements. If you take redux, they have a single one: https://react-redux.js.org/api/hooks#useselector.

Reply:
in hindsight, I even wonder if I'm not concens more by having selector like `gridPreferencePanelStateSelector` instead of duplicating them everywhere, which seems a lot better.